CWE/events/albania.txt

```diff
@@ -242,3 +242,112 @@ country_event = {
 		ai_chance = { factor = 0.1 }
 	}
 }
+
+country_event = {
+	id = 8022808
+	title = "$YEAR$ Albanian Parliamentary Elections"
+	desc = EVTDESC8022808
+	picture = "transition_albania"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = ALB
+		has_global_flag = 1992_start_date_flag
+
+
+		government = socialist_democracy
+		ruling_party_ideology = socialist
+		war = no
+		is_vassal = no
+		
+	}
+
+	mean_time_to_happen = { 
+		months = 2
+	}
+
+	option = {
+		name = "A Democratic Party victory sweeps away the remaining Socialist policies."
+		
+		government = democracy
+		country_event = 800054
+		ruling_party_ideology = conservative
+
+		any_pop = { 
+			limit = { strata = rich }
+			ideology = { factor = 0.2 value = conservative }
+		}
+		any_pop = { 
+			limit = { strata = middle }
+			ideology = { factor = 0.1 value = liberal }
+			ideology = { factor = 0.1 value = conservative }
+		}
+		any_pop = { 
+			limit = { strata = poor }
+			ideology = { factor = 0.1 value = socialist }
+			ideology = { factor = 0.1 value = populist }
+			ideology = { factor = 0.1 value = communist_social }
+		}
+		
+		any_pop = {
+			scaled_militancy = { 
+				factor = 10
+				ideology = communist
+			}
+			scaled_militancy = { 
+				factor = 10
+				ideology = communist_social
+			}
+			scaled_militancy = { 
+				factor = 5 
+				ideology = socialist
+			}
+		}
+		
+		ai_chance = { factor = 0.7 }
```

Comment:
Historical option should have a 90% ai chance.

Reply:
Alright.